.github/workflows/linux-wpt.yml

```diff
@@ -39,14 +39,23 @@ jobs:
         with:
           name: release-binary
           path: release-binary
+      - name: Cache libffi
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: ./libffi6_3.2.1-8_amd64.deb
+          key: cache-libffi
```

Comment:
It'd probably be a good idea to encode the version into this key so that new versions are a cache miss.

Reply:
Good point. This was a quick fix. Ideally the key & package url/name will be derived from a single source (e.g variable defintion, or from file) so we can updating one place will update both key and package url.

In the long term, I guess we'll need to know what causes the dependency on ffi6 so that we can upgrade that to use the system libffi7.

Reply:
I think that this dependency on an old version of libffi is caused by the very old version of GStreamer that we ship with our binaries.

Reply:
That makes sense. Should we consider shipping newer GStreamer on Linux? I believe we ship 1.22.3 on MacOS.

Is it possible the support for a minimum of 1.16 of Gstreamer required for some particular compatiblity reason, like supporting Servo on Android? Or we just simply never updated them?